### ISSUE TITLE
1.17.0 broke compatibility with ruby <2.7

### ISSUE THREAD

**Issue by gavinsharp** [ruby-git/ruby-git#648]
### Subject of the issue
The addition of the `activesupport` dependency in 1.17.0 (https://github.com/ruby-git/ruby-git/pull/645) broke us (via [`danger` v8.4.*](https://github.com/danger/danger/blob/314bb927605beb862137498e36a3b4bfff413dc7/danger.gemspec#L25)):
```
ERROR:  Error installing danger:
	The last version of activesupport (>= 4.0.0) to support your Ruby & RubyGems was 6.1.7.2. Try installing it with `gem install activesupport -v 6.1.7.2` and then running the current command again
	activesupport requires Ruby version >= 2.7.0. The current ruby version is 2.6.10.210.
```

This is probably desirable in a sense (see https://github.com/ruby-git/ruby-git/issues/550), but it seems like it would merit a major version bump?

### Your environment
* version of ruby: 2.6

### Steps to reproduce
Try to install `danger` v8.6 using Ruby 2.6, which has a runtime dependency on `git ~> 1.7`.

### Expected behaviour
Success

### Actual behaviour
Failure

**Comment by jcouball** [comment]
I will revert the change which brought in ActiveSupport and tag it for the next major release which will also upgrade the minimum dependencies for Ruby and git. In the meantime, can you pin yourself to 1.16.0?

**Comment by gavinsharp** [comment]
> In the meantime, can you pin yourself to 1.16.0?

We actually just upgraded to ruby 2.7, was not a big deal for our use case (running Danger in CI). Just thought I would flag the semver implications since the breakage was unexpected (and could potentially affect others in more serious ways).

**Comment by jcouball** [comment]
Fixed in v1.7.1

The Ruby dependency change was inadvertent, but something I plan on doing very soon anyway.

FWIW, since Ruby MRI 2.7.x will not receive updates after March 30th, 2023, I was planning on bumping the dependency of this gem to 3.0.x. I am curious on your thoughts of that.

**Comment by gavinsharp** [comment]
> FWIW, since Ruby MRI 2.7.x will not receive updates after March 30th, 2023, I was planning on bumping the dependency of this gem to 3.0.x. I am curious on your thoughts of that.

We only rely on this gem via Danger, so I think that wouldn't be an issue for us (a quick try with ruby 3.0 or 3.2 seems to cause no issues).
